bot/bot_test.ttl

```diff
@@ -0,0 +1,114 @@
+@prefix : <http://bot_test#> .
+@prefix cc: <http://creativecommons.org/ns#> .
+@prefix bot: <https://w3id.org/bot#> .
+@prefix dbo: <http://dbpedia.org/ontology/> .
+@prefix dce: <http://purl.org/dc/elements/1.1/> .
+@prefix owl: <http://www.w3.org/2002/07/owl#> .
+@prefix rdf: <http://www.w3.org/1999/02/22-rdf-syntax-ns#> .
+@prefix xml: <http://www.w3.org/XML/1998/namespace> .
+@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
+@prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
+@prefix vann: <http://purl.org/vocab/vann/> .
+@prefix bot_test: <http://bot_test#> .
+@base <http://bot_test> .
+
+<http://bot_test> rdf:type owl:Ontology ;
+                   owl:imports <https://w3id.org/bot> ;
+                   dce:description "Example showing how the BOT ontology can be used to describe the topology of a building.
+The building has four external walls and consists of 2 storeys. In the first storey there are rooms 101 and 102. In the second storey there is room 201.
+Room 101 has an entrance door, one window and a table.
+Room 102 has a window and a sofa.
+A door connects room 101 and room 102.
+A trapdoor connects room 102 and room 201." ;
+                   dce:creator "Walter Terkaj (walter.terkaj@itia.cnr.it)" ;
```

Comment:
can use a full URI (some URL that points to a foaf description of you on your website ?)